### ISSUE TITLE
weird rounded tabs on linux or with persona

### ISSUE THREAD

**Issue by davidbepo** [axydavid/FirefoxUI#4]
![captura de pantalla de 2017-12-03 10-54-21](https://user-images.githubusercontent.com/18667173/33524222-91b3e316-d818-11e7-99d6-e10ef42fde97.png)

**Comment by davidbepo** [comment]
also a close button for the other tabs without having to switch to them would be nice

**Comment by axydavid** [comment]
1. Uncomment the /* NIGHTLY FIX */ section of the css code(line 11 trough 20), that should fix the tabs issue.

2. I only have the close button on the active tab,  I'm trying to have default settings that are viable for all users, remove line 8 and you should have the close button for all tabs.

**Comment by davidbepo** [comment]
the issue happens even with that lines uncommented

**Comment by davidbepo** [comment]
and thanks for the close button thing

**Comment by axydavid** [comment]
Unfortunately I don't have linux so it's hard for me to know how to fix it.
However as far as I can see the issue is that the tab height is too big.
I'll try to get a virtual machine running with linux.
Try to run the stable branch of firefox and see if you get the same issue.

**Comment by GreenWoodie** [comment]
I don't think this is necessarily a Linux issue. I had the same problem on Firefox v56 running on Windows 10  but I'd been playing around with some of your code so couldn't be absolutely sure if it was you or me. Anyway It cleared up with v57. Not sure if that helps?

**Comment by davidbepo** [comment]
even on windows it doesnt look good with a persona:
![sin titulo](https://user-images.githubusercontent.com/18667173/33737851-d5b29c38-db97-11e7-95a6-4ad51d7133f5.jpg)
I wouldn't call that a Issue tho.....as I clearly state in the main page that you should run the default skin.
There MIGHT be some hope of fixing the color issue, but I don't promise anything.
As for now, just try firefox v57 as @GreenWoodie said and see if that fixes your issues, unfortunatelly it's really hard to support like 4 different versions of Firefox, as sometimes Mozilla tends to make small changes in the browser that messes up my css code...
I personally run nightly, so at least I know it works in nightly.
